packages/context-menu/src/vaadin-contextmenu-items-mixin.js

```diff
@@ -3,37 +3,10 @@
  * Copyright (c) 2016 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
+import './vaadin-context-menu-item.js';
```

Comment:
Should these imports be added to `vaadin-contextmenu-items-mixin.d.ts`?

Reply:
Maybe they should as it would probably help TS to pick up custom elements definitions. Added them.